Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -471,6 +471,9 @@ void ProgramShaderCache::Shutdown()
 	// store all shaders in cache on disk
 	if (g_ogl_config.bSupportsGLSLCache && !g_Config.bEnableShaderDebugging)
 	{
+		// Clear any prior error code
+		glGetError();
```

Comment:
I don't think that would be the case, we already have a `glGetError()` call at the end of every iteration. Why would it need to be cleared again?